eng/common/TestResources/New-TestResources.ps1

```diff
@@ -469,20 +471,20 @@ try {
             }
         }
 
-        $preDeploymentScript = $templateFile | Split-Path | Join-Path -ChildPath 'test-resources-pre.ps1'
+        $preDeploymentScript = $templateFile.originalFilePath | Split-Path | Join-Path -ChildPath 'test-resources-pre.ps1'
         if (Test-Path $preDeploymentScript) {
             Log "Invoking pre-deployment script '$preDeploymentScript'"
             &$preDeploymentScript -ResourceGroupName $ResourceGroupName @PSBoundParameters
         }
 
-        Log "Deploying template '$templateFile' to resource group '$($resourceGroup.ResourceGroupName)'"
+        Log "Deploying template '$($templateFile.jsonFilePath)' to resource group '$($resourceGroup.ResourceGroupName)'"
```

Comment:
IMO, this should be the `originalFilePath` since bicep would've failed to compile and broken the script earlier.

Reply:
Alternatively, you could do something like:

```powershell
$msg = if ($templateFile.jsonFilePath -ne $templateFile.originalFilePath) {
  "Deployment template '$($templateFile.jsonFilePath)' ($templateFile.originalFilePath) to resource group ..." } else {
  "deployment template '$($templateFile.jsonFilePath)' to resource group ..."
}
```

Reply:
@heaths why do you think this is an error case where the bicep compile failed?

Reply:
If the bicep file has a syntax error, a listKeys* function is wrong (I did that once, and while the bicep file compiled the deployment error was misleading), etc. Plenty of reasons user input may fail to compile.

Reply:
I understand that the bicep build might fail but I'm not sure what this message has to do with that failure case.

Reply:
I never said anything about a failure case. I saw I think the message should point to the original file in some way. *IIF* the bicep compilation failed, we never would've gotten that far anyway so it doesn't matter.

Reply:
OK I think I misunderstood. I'm good either way.